pkgs/build-support/docker/default.nix

```diff
@@ -209,7 +209,7 @@ rec {
 
       postMount = ''
         echo "Packing raw image..."
-        tar -C mnt --mtime=0 -cf $out .
+        tar -C mnt --mtime='@1' -cf $out .
```

Comment:
I chose `'@1'` since I duplicated the method I found at [nixos/lib/make-system-tarball.sh line 57](https://github.com/NixOS/nixpkgs/blob/1b1b9816fc0d90d835661c845bd8ae041290c087/nixos/lib/make-system-tarball.sh#L57).  I have nothing against using `'@0'` if desired.

Perhaps I didn't go far enough and should have also included `--sort=name`?